pep-9999.rst

```diff
@@ -261,15 +261,19 @@ type is assumed to be ``Any``. It is preferred to leave unknown
 types unannotated rather than explicitly marking them as ``Any``, as some
 type checkers can optionally warn about unannotated arguments.
 
-The type of
-an argument where the annotation and the type of the default argument disagree
-is unspecified. As an exception, the ellipsis literal can stand in for any
-type::
-
-    # x has type Any
-    # y's type is unspecified
-    # z has type int
-    def foo(x, y: int = "", z: int = ...): ...
+If an argument has a default value, it must match the implementation
```

Comment:
It's legal (if unadvisable) to do something like this in your code:
```
class Foo:
  pass
def f(x: Foo = Foo()): ...
```
I don't think we'd want this in a stub, since it would require evaluating `Foo()` to determine what the default value is. Should we specify that default values in stubs should be constants?